src/util.ts

```diff
@@ -80,7 +80,7 @@ export class GoVersion {
 
 	constructor(version: string) {
 		const matchesRelease = /go version go(\d.\d+).*/.exec(version);
-		const matchesDevel = /go version devel \+(.\d+).*/.exec(version);
+		const matchesDevel = /go version devel \+(.[a-zA-Z0-9]+).*/.exec(version);
```

Comment:
What does this fix?

Reply:
Go developer versions have a Git commit hash in them, for example:

`go version devel +868de9a111 Tue Oct 8 17:30:14 2019 +0000 linux/amd64`

This contains not just numbers, but letters as well.